docs/INSTALLATION.md

```diff
@@ -1,65 +1,90 @@
 # Installation
 
-## Installing Stack
 
-If you don't have a recent Stack version installed in your system, follow the
-[installation instructions](http://docs.haskellstack.org/en/stable/install_and_upgrade/)
-for your operating system. After completing the installation, test that you
-have installed it correctly by running this command in a terminal:
+## Get GHCup
 
-```bash
-stack --version
-```
+The recommended way of installing Haskell tools is through [GHCup][ghcup].
```

Comment:
We might also reference where this information comes from, the recommendation stated at https://www.haskell.org/downloads/

This can mean that when updates and changes happen, we are less likely to be outdated.